format/view-spec.md

```diff
@@ -42,12 +42,24 @@ An atomic swap of one view metadata file for another provides the basis for maki
 
 Writers create view metadata files optimistically, assuming that the current metadata location will not be changed before the writer's commit. Once a writer has created an update, it commits by swapping the view's metadata file pointer from the base location to the new location.
 
+### Materialized Views
+
+Views have the option to be turned into materialized views by precomputing the data from the view query.
+When queried, materialized views return the precomputed data, shifting the cost of query execution to the precomputation step.
+
+Iceberg materialized views are implemented as a combination of an Iceberg view and an underlying Iceberg table, known as the storage table, which stores the precomputed data.
+The metadata for a materialized view extends the common view metadata, adding a pointer to the precomputed data and freshness information to determine if the data is still fresh. 
+The storage table can have the states "fresh", "stale" or "invalid".
```

Comment:
I don't see a property for this state on the storage table?   I also don't think we need it.

Reply:
What I mean is that if you check for the freshness of the table, the result can be "fresh", "stale", or "invalid". Maybe state is not the right term for it. And you can't assign a property to it. Maybe we can drop the sentence from the spec.

Reply:
oic now, then I don't think we need to drop the sentence.

Reply:
maybe a minor change from `can have` to `can be` would avoid the misinterpretation. `have` can imply `containing the state field`.